composer.json

```diff
@@ -9,6 +9,11 @@
       "Outlandish\\Wordpress\\Routemaster\\": "src/"
     }
   },
+  "require": {
+    "php": ">=5.6",
+    "ext-json": "*",
+    "outlandish/oowp": "2.*"
```

Comment:
Not sure this is really a requirement. We built the two plugins with independence in mind, but maybe we've put in some inadvertent dependencies

Reply:
There seem to be several pretty hard dependencies at the moment @rasmuswinter, e.g.

```
use Outlandish\Wordpress\Oowp\Views\PostView;

class ContainerView extends PostView
{
  ...
}
```

If we want them decoupled I guess we need a shared lib with some interfaces – but for now I think this reflects the reality.

Reply:
Oh, but these are isolated to `Outlandish\Wordpress\Routemaster\Oowp`. Maybe we could downgrade it to a `suggests` and explain what's going on here in the readme for now?

I think splitting this folder out to an optional lib eventually is probably cleanest, so there aren't classes used in any core code that may or may not be available, and so you aren't shipped code you don't need if you don't use `oowp`.

Reply:
Those dependencies should be restricted to the `oowp` namespace of routemaster, which is intended to only be used if the `oowp` plugin is present. 

I guess what I should have said is that routemaster *can* work without `oowp`, but there's enhanced functionality with it

Reply:
Thanks @rasmuswinter, I've used `suggest` now and added to the readme.